### PR TITLE
Added Codeberg and Papara

### PR DESCRIPTION
**Pull request by ElongatedVeggie**
Papara is mainly a Turkish service so please feel free to not include it if you feel like it doesn't suit the project. Also, I was planning to do one more Turkish service that starts with "Ç". Where would that be in the alpabetical listing, if it is even allowed? Thank you.

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
> Papara is mainly a Turkish service so please feel free to not include it if you feel like it doesn't suit the project.

We accept "local" services too, after all we want to help as much people as possible, so don't worry.

> Also, I was planning to do one more Turkish service that starts with "Ç". Where would that be in the alpabetical listing, if it is even allowed? Thank you.

It is, but the sorting uses "ASCII" sorting, so it would be the last item on the list, if it starts with Ç :D

File: _data/sites.json [hunk 1]
```diff
@@ -7194,6 +7204,17 @@
...
     },
 
+    {
+        "name": "Papara",
+        "url": "https://www.papara.com/",
+        "difficulty": "hard",
+        "notes": "You need to call the customer support in order to get your account deleted.",
+        "notes_tr": "Hesabınızı kapatmak için müşteri hizmetleriyle telefon ile iletişime geçin.",
+        "domains": [
+            "www.papara.com"
```
Comment (tupaschoal) [inline _data/sites.json:7214]:
```suggestion
            "papara.com"
```

**Comment by ElongatedVeggie** [comment]
While we're at it, giving phone numbers of support lines is not allowed, is it?

**Comment by tupaschoal** [comment]
> While we're at it, giving phone numbers of support lines is not allowed, is it?

I'd say no one ever tried. I don't see why it wouldn't be valid, though I probably would take a link with the phone being quoted to make sure it seems legit.

**Comment by tupaschoal** [comment]
Thanks for your contribution! I've merged this one, feel free to open more PRs with the next ones

**Pull request merged**